src/main/java/com/urbanairship/connect/client/StreamConsumeTask.java

```diff
@@ -118,7 +118,7 @@ private void stream() throws ConnectionException {
 
     private Optional<StartPosition> getPosition() {
         Optional<String> lastOffset = consumer.get();
-        log.debug("Position - Consumer: " + consumer + " InitialPosition: " + initialPosition + " Last Offset: " + lastOffset);
+        log.debug("Consumer last offset: " + consumer.lastOffset + ", InitialPosition: " + initialPosition + ", Last Offset: " + lastOffset);
```

Comment:
how should a person interpret consumer last offset vs. last offset?

Reply:
removed the last offset. The consumer offset and the initial position here should be enough.